test/test_parabolic_1d.jl

```diff
@@ -28,6 +28,41 @@ isdir(outdir) && rm(outdir, recursive = true)
     end
 end
 
+@trixi_testset "TreeMesh1D: elixir_advection_diffusion_ldg.jl" begin
+    # We use a stricter time integration tolerance since there are
+    # minor difference in CI otherwise.
+    @test_trixi_include(joinpath(examples_dir(), "tree_1d_dgsem",
+                                 "elixir_advection_diffusion_ldg.jl"),
+                        initial_refinement_level=4, tspan=(0.0, 0.4), polydeg=3,
+                        l2=[9.234438322025545e-6], linf=[5.4254945236031915e-5])
+    # Ensure that we do not have excessive memory allocations
+    # (e.g., from type instabilities)
+    let
+        t = sol.t[end]
+        u_ode = sol.u[end]
+        du_ode = similar(u_ode)
+        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000
+    end
+end
+
+@trixi_testset "TreeMesh1D: elixir_advection_diffusion_ldg.jl convergence" begin
```

Comment:
```suggestion
@timed_testset "TreeMesh1D: elixir_advection_diffusion_ldg.jl convergence" begin
```
I'm not sure, but could you try this?

Reply:
Thanks! Fingers crossed